packages/@aws-cdk/aws-lambda/lib/runtime.ts

```diff
@@ -34,7 +34,7 @@ export class Runtime {
   public static readonly NODEJS_6_10 =    new Runtime('nodejs6.10',     RuntimeFamily.NODEJS, { supportsInlineCode: true });
   /** @deprecated Use {@link NODEJS_10_X} */
   public static readonly NODEJS_8_10 =    new Runtime('nodejs8.10',     RuntimeFamily.NODEJS, { supportsInlineCode: true });
-  public static readonly NODEJS_10_X =    new Runtime('nodejs10.x',     RuntimeFamily.NODEJS, { supportsInlineCode: false });
+  public static readonly NODEJS_10_X =    new Runtime('nodejs10.x',     RuntimeFamily.NODEJS, { supportsInlineCode: true });
```

Comment:
does node10 support inlining? is this new?

Reply:
Seems like it, but I haven't seen any official confirmation yet.

* https://github.com/aws/aws-cdk/issues/4642#issue-511213472
  > I have checked this with AWS Support. They're not familiar with the CDK, but doing the same thing in pure CloudFormation works with both 8 and 10
* https://github.com/aws/aws-cdk/commit/3af61c7e971d0a0ef425eab241c824dd0a9e79d6 built successfully

Reply:
The fact that it builds doesn't mean that it works in CF (integ tests are not run).

Just ran an integ test in `eu-west-1` and got the following error message:
> ZipFile can only be used when Runtime is set to either of nodejs, nodejs4.3, nodejs6.10, nodejs8.10, python2.7, python3.6, python3.7.

Reply:
Whoops, thanks @jogold. I thought they were ran.